app/marketing/management/commands/expiration_start_work.py

```diff
@@ -25,8 +25,12 @@
 from django.utils import timezone
 
 from dashboard.models import Bounty, Interest
+from dashboard.notifications import maybe_notify_user_removed_github, maybe_warn_user_removed_github
 from github.utils import get_interested_actions
-from marketing.mails import bounty_startwork_expire_warning, bounty_startwork_expired
+from marketing.mails import (
+    bounty_startwork_expire_warning, bounty_startwork_expired, maybe_notify_bounty_user_removed_to_slack,
```

Comment:
This import will fail. `maybe_notify_bounty_user_removed_to_slack` and `maybe_notify_bounty_user_warned_removed_to_slack`